app/stac_api/models.py

```diff
@@ -328,7 +328,8 @@ def from_db(cls, db, field_names, values):
         return instance
 
     def __str__(self):
-        return self.name
+        # This is used in the admin page in the autocomplete_fields of the Asset page
+        return f"{self.collection.name}/{self.name}"
```

Comment:
Wouldn't it make sense to also adapt the asset's `__str__(self)`, too?

Reply:
For consistency could be, but otherwise I don't see any use case where it would be useful. Other than for item I had to add a `item_name` method in the `AssetAdmin` in order to have only the item name without the collection name due to this `Item.__str__(self)`.